src/routes/transactions.ts

```diff
@@ -45,7 +47,7 @@ export const createSepaDirectDebit = async (req, res) => {
   const technicalPerson = await getTechnicalUserPerson();
```

Comment:
findPersonByAccountIBAN is quite slow I think, it does this super slow `getAllPersons` under the hood

Reply:
it gets person by id. It's not using `getAllPersons`